bin/test-workflow/start

```diff
@@ -3,16 +3,108 @@
 set -eo pipefail
 cd "$(dirname "$0")" || ( echo "cannot cd into dir" && exit 1 )
 
-./0_prep_conjur_in_kind.sh
+function print_help {
+  cat << EOF
+Runs the end-to-end workflow for testing Conjur Kubernetes authentication.
+By default, the workflow is run against Conjur Open Source, but can be run
+against Conjur Enterprise.
 
-source ./1_prep_env.sh
+Usage: ./start [options]:
 
-./2_admin_load_conjur_policies.sh
-./3_admin_init_conjur_cert_authority.sh
+    -o, --open-source         Run the E2E workflow against Conjur Open Source
+    -e, --enterprise          Run the E2E workflow against Conjur Enterprise
+    -p, --platform <pform>    Platform on which to delpoy Conjur
+                              Flag required for Enterprise workflow
+                              Only accepts 'kind' for Open Source workflow
```

Comment:
It would be possible to default the `--platform` dependent on whether the Open Source workflow vs Enterprise workflow is selected. The description would be something like:
```
                              For Open Source workflow:
                                   - Defaults to 'kind'
                                   - All other selections are rejected
                              For Enterprise workflow:
                                   - Defaults to 'gke'
                                   - All other selections are rejected
```

Reply:
The default behavior does currently work how you describe - I left the flag so that eventually it could accept `oc` as well.